js/Grid/Grouping.md

```diff
@@ -61,7 +61,7 @@ $(function () {
 
 The following output is displayed as a result of the above code example.
 
-![](Grouping_images/Grouping_img2.png)
+![Initial grouping in Angular Grid](Grouping_images/javascript-grid-initial.png)
```

Comment:
Platform name is wrong, check and change.

Reply:
Done